go/apps/simpleexample/SimpleClientCpp/helpers.cc

```diff
@@ -26,7 +26,7 @@
 #include <string>
 #include <thread>
 
-#include <taosupport.pb.h>
```

Comment:
I almost did that originally but I'm not sure it's worth separating the app across the directories.  Where would SimpleDomain go?